packages/schematics/angular/utility/ast-utils_spec.ts

```diff
@@ -685,4 +687,164 @@ describe('ast utils', () => {
       });
     });
   });
+
+  describe('insertImport', () => {
```

Comment:
These are tests mostly for existing functionality. I added them since we had no coverage.